src/interaction.py

```diff
@@ -189,9 +191,11 @@ def callM(mvals_, dielectric=False):
         D = emm - self.eqm
         dV = np.zeros((self.FF.np,len(emm)))
 
-        # Dump interaction energies to disk.
-        np.savetxt('M.txt',emm)
-        np.savetxt('Q.txt',self.eqm)
+        if self.writelevel > 0:
```

Comment:
I agree. When `writelevel` is specified, the plotting function will depend on matplotlib to function.